src/providers/object-provider.js

```diff
@@ -1,6 +1,7 @@
 import {
     qualifiedNameToId,
-    accumulateResults
+    accumulateResults,
+    makeKeyString
```

Comment:
The Open MCT object api exposes this function, so we could avoid duplicating this function in this repo by just passing the open mct instance into the constructor for `YamcsObjectProvider`

Reply:
Dang! I thought so too, shoulda said something : )

Reply:
done. I had thought so too.... and then didn't : (